app/assets/javascripts/users.coffee

```diff
@@ -13,4 +13,18 @@ ready =->
     event.preventDefault()
     $('.account-link-description').toggle()
 
-$(document).on('turbolinks:load', ready)
\ No newline at end of file
+  $('.delete-user').on 'click', (event) ->
+    event.preventDefault()
+    $('#notification-modal').modal()
+
+deleteUser = (deletePath) ->
+  if confirm(I18n.t('users.show.delete_modal.last_confirm'))
+    post(deletePath, {_method: 'delete', authenticity_token: $('meta[name=csrf-token]').attr('content')})
+  else
+    $('#notification-modal').modal('hide')
+
+
+$(document).on('turbolinks:load', ready)
+
+root = exports ? this;
```

Comment:
What is the `exports` used for here? I only know it from module exports, but this is none, is it?

Reply:
it puts the method in a global namespace, so I can call it from html - I'm not sure where I got it from